PayPalMessaging/build.gradle

```diff
@@ -27,12 +36,6 @@ android {
         }
     }
 
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
```

Comment:
Was this causing problems too?

Reply:
It wasn't causing problems, but I don't think it's needed. None of the other modules have this defined and we don't have a proguard rules file for this module.